src/components/CheckIn.js

```diff
@@ -34,15 +33,19 @@ export class CheckInForm extends Component {
       values.weight = parseInt(values.weight) || null;
       const person = {...values, picture : this.picture};
       var matches = [];
+
+      const potentialMatches = checkInAlgorithm(person, { limit: 20 });
```

Comment:
should be formatted `const potentialMatches = (checkInAlgorithm(person), { limit: 20 });`

Reply:
Are the wrapping `()` around the function necessary here?